settings.php

```diff
@@ -307,10 +307,23 @@
     $setting->set_updatedcallback('theme_reset_all_caches');
     $page->add($setting);
 
+    // Setting to display the add a block widget as in classic Boost.
+    $name = 'theme_boost_campus/settingsaddablockposition';
+    $title = get_string('settingsaddablockposition', 'theme_boost_campus', null, true);
+    $description = get_string('settingsaddablockposition_desc', 'theme_boost_campus', null, true);
+    $settingsaddablockpositionsetting = [
```

Comment:
I'm wondering why you are using yes and no here instead of describing names (compare https://github.com/moodleuulm/moodle-theme_boost_campus/blob/a7024d673421c858b45cd1817341048194e542fc/settings.php#L329).
Please change this to positionnavdrawer and positionblockregion, for example.
And don't forget to change the related usages.